doc/changelog.rst

```diff
@@ -100,6 +100,9 @@ Breaking Changes in 4.0
 - Removed :class:`bson.binary.UUIDLegacy`.
 - The "tls" install extra is no longer necessary or supported and will be
   ignored by pip.
+- ``tz_aware``, a keyword argument for :class:`~pymongo.mongo_client
+.MongoClient`, now defaults to ``False`` everywhere instead of ``True`` in
```

Comment:
Here and elsewhere, the `:class:...` directive has to be all on one line or else it won't render correctly. Check out the link to the rendered docs in the PR "checks" dropdown.

Reply:
Done.